diff --git a/node_modules/openapi-client-axios/client.js b/node_modules/openapi-client-axios/client.js
index 8b156bd..b0bf34a 100644
--- a/node_modules/openapi-client-axios/client.js
+++ b/node_modules/openapi-client-axios/client.js
@@ -378,9 +378,8 @@ var OpenAPIClientAxios = /** @class */ (function () {
             // construct axios request config
             var axiosConfig = {
                 method: request.method,
-                url: request.path,
+                url: request.url,
                 data: request.payload,
-                params: request.query,
                 headers: request.headers,
             };
             // allow overriding baseURL with operation / path specific servers
